test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -66,6 +66,15 @@ class test_ucp_sockaddr : public ucp_test {
         m_err_count = 0;
         modify_config("KEEPALIVE_INTERVAL", "10s");
         modify_config("CM_USE_ALL_DEVICES", cm_use_all_devices() ? "y" : "n");
+
+        /*
+         * FIXME: this is a workaround of the issue reproduced by
+         *        the 'close_ep_force_before_err_cb' test with RC transport
+         *        where TX-queue less than FC_WND, so uct_flush cancel returns
+         *        UCS_ERR_NO_RESOURCES and can not be handled after error
+         */
+        modify_config("RC_FC_WND_SIZE", "128", SETENV_IF_NOT_EXIST); 
```

Comment:
@dmitrygx can it be related to MTT issue with FC?

Reply:
but we don't do flush-cancel there

Reply:
ucp_ep_close(force) does

Reply:
> ucp_ep_close(force) does

yes, but we don't receive FC_PURE_GRANT by some reason in normal flow

Reply:
then it seems unrelated